app/templates/tasks.hbs

```diff
@@ -13,7 +13,7 @@
         Organizations
       </p>
       <ul class="menu-list">
-        {{#each-in organizations.list as |slug org| }}
+        {{#each-in organizations.list as |slug org|}}
```

Comment:
this should be fix in the first PR, if it is important
see https://github.com/coala/git-task-list/pull/33#discussion_r194412825